### PR TITLE
prepare connector for `2201.0.0` release

### PR DESCRIPTION
**Pull request by SkNuwanTissera**
# Description

prepare connector for `2201.0.0` release

### Security checks
 - [X] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? Yes
 - [X] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? Yes

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,3 +1,4 @@
 org.gradle.caching=true
 group=org.ballerinalang.microsoft.teams
-version=2.1.0
+version=2.2.0
+ballerinaLangVersion=2201.0.0
```
Comment (LakshanSS) [inline gradle.properties:4]: Shall we remove this `ballerinaLangVersion`?

We purposefully removed it earlier.

if we have java-wrappers only, we need to include this.

Comment (SkNuwanTissera) [reply gradle.properties:4]: Yeah. But without this we cannot build the code. 
```
> Could not get unknown property 'ballerinaLangVersion' for root project 'module-ballerinax-microsoft.teams' of type org.gradle.api.Project.
```

Comment (SkNuwanTissera) [reply gradle.properties]: Removed as suggested.

Comment (LakshanSS) [reply gradle.properties]: Need to remove the usage of `ballerinaLangVersion` also in the project.

**Review by LakshanSS — APPROVED** [review]

**Comment by SkNuwanTissera** [comment]
https://github.com/ballerina-platform/ballerina-extended-library/issues/272

**Pull request merged**